ibc-testkit/src/testapp/ibc/utils.rs

```diff
@@ -0,0 +1,36 @@
+use core::time::Duration;
+
+use ibc::core::client::types::Height;
+use ibc::primitives::Timestamp;
+use ibc_proto::google::protobuf::{Duration as GDuration, Timestamp as GTimestamp};
+
+pub fn timestamp_gpb_to_ibc(gpb_timestamp: GTimestamp) -> Timestamp {
+    let GTimestamp { seconds, nanos } = gpb_timestamp;
+    Timestamp::from_nanoseconds(seconds as u64 * 1_000_000_000 + nanos as u64)
+        .expect("no hmm overflow")
+}
+
+pub fn timestamp_ibc_to_gpb(ibc_timestamp: Timestamp) -> GTimestamp {
+    let tendermint_proto::google::protobuf::Timestamp { seconds, nanos } = ibc_timestamp
```

Comment:
actually this is needed because I am using `tendermint::Time` to `tendermint_proto::google::protobuf::Timestamp` and then to `ibc_primitives::types::Timestamp` :sweat_smile:

I know we can avoid `tendermint_proto::google::protobuf::Timestamp` by doing the conversion manually.

Reply:
removed these functions as we started to use u64 to represent nanoseconds.